### PR TITLE
Test/fix unstable

### PR DESCRIPTION
**Pull request by andriidovhan**
- increase waitTime in waitPortIsStable(for a porn with non-direct link)
- add discoveryInterval in DefaultRulesSpec
- refactor "Can create mirror point on a VXLAN flow, #mirrorDirection" 
- replace "antiflap" with "nb" in tests with free-isl port
- refactor Server42FlowRttSpec 
- refactor "Cannot enable connected devices on switch if mirror is present"  (run in multiTable mode only)

### CODE REVIEW

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/AutoRerouteSpec.groovy [hunk 1]
```diff
@@ -291,14 +291,14 @@ class AutoRerouteSpec extends HealthCheckSpecification {
...
-        antiflap.portDown(flow.source.datapath, flow.source.portNumber)
+        northbound.portDown(flow.source.datapath, flow.source.portNumber)
 
         then: "The flow is not rerouted"
         TimeUnit.SECONDS.sleep(rerouteDelay)
         PathHelper.convert(northbound.getFlowPath(flow.id)) == flowPath
 
         when: "Bring the flow port down on the destination switch"
-        antiflap.portDown(flow.destination.datapath, flow.destination.portNumber)
+        northbound.portDown(flow.destination.datapath, flow.destination.portNumber)
```
Comment (rtretyak) [inline src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/AutoRerouteSpec.groovy:301]: you can't do northbound port down here and do antiflap port up in cleanup. Makes no sense. Also, why this change happen at the first place?

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/AutoRerouteV2Spec.groovy [hunk 2]
```diff
@@ -552,14 +552,14 @@ class AutoRerouteV2Spec extends HealthCheckSpecification {
...
-        antiflap.portDown(flow.source.switchId, flow.source.portNumber)
+        northbound.portDown(flow.source.switchId, flow.source.portNumber)
 
         then: "The flow is not rerouted"
         TimeUnit.SECONDS.sleep(rerouteDelay)
         PathHelper.convert(northbound.getFlowPath(flow.flowId)) == flowPath
 
         when: "Bring the flow port down on the destination switch"
-        antiflap.portDown(flow.destination.switchId, flow.destination.portNumber)
+        northbound.portDown(flow.destination.switchId, flow.destination.portNumber)
```
Comment (rtretyak) [inline src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/AutoRerouteV2Spec.groovy:562]: same

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/server42/Server42FlowRttSpec.groovy [hunk 3]
```diff
@@ -887,12 +887,12 @@ class Server42FlowRttSpec extends HealthCheckSpecification {
     def changeFlowRttSwitch(Switch sw, boolean requiredState) {
         def originalProps = northbound.getSwitchProperties(sw.dpId)
         if (originalProps.server42FlowRtt != requiredState) {
+            def s42Config = topology.switches.find { sw.dpId == it.dpId}.prop
```
Comment (rtretyak) [inline src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/server42/Server42FlowRttSpec.groovy:890]: just `sw.prop` :)

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/SwitchPortConfigSpec.groovy [hunk 4]
```diff
@@ -91,13 +91,13 @@ class SwitchPortConfigSpec extends HealthCheckSpecification {
...
         def port = topology.getAllowedPortsForSwitch(sw).find { "LINK_DOWN" in northbound.getPort(sw.dpId, it).state }
-        antiflap.portDown(sw.dpId, port)
+        northbound.portDown(sw.dpId, port)
 
         then: "Port is really DOWN"
         Wrappers.wait(WAIT_OFFSET) { assert "PORT_DOWN" in northbound.getPort(sw.dpId, port).config }
 
         when: "Bring port up on the switch"
-        def portUp = antiflap.portUp(sw.dpId, port)
+        def portUp = northbound.portUp(sw.dpId, port)
```
Comment (rtretyak) [inline src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/SwitchPortConfigSpec.groovy:100]: cleanup should be fixed as well. Also don't really understand why this change is required

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/server42/Server42FlowRttSpec.groovy (hunk 3 above)
Comment (andriidovhan) [reply src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/server42/Server42FlowRttSpec.groovy]: hah, fixed)
